src/index.tsx

```diff
@@ -119,12 +119,12 @@ function Dashboard() {
     const { t } = useI18N()
     const classes = useStyles()
 
-    const shouldRenderAppBar = webpackEnv.firefoxVariant === 'GeckoView' || webpackEnv.target === 'WKWebview'
+    const shouldRenderAppBar = webpackEnv.genericTarget === 'app'
     const shouldNotRenderAppBar = useMediaQuery('(min-width:1024px)')
 
     const routers: [string, string, JSX.Element][] = [
         [t('home'), '/home/', <HomeIcon />],
-        ['Wallets', '/wallets/', <CreditCardIcon />],
+        ...((webpackEnv.genericTarget === 'app' ? [] : [['Wallets', '/wallets/', <CreditCardIcon />]]) as any),
```

Comment:
Does it mean it can't use the wallet to receive a red packet on the android app?